core/templates/dev/head/services/contextual/LoggerService.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
```

Comment:
Are we going to follow the same file naming convention?

Reply:
Shouldn't we follow the same convention?

Reply:
Yeah, I think we have an issue for the same at #6975. (Can you please confirm and do the changes if required?)

Reply:
@bansalnitish is this resolved?

Reply:
Done, ptal